Formula/fio.rb

```diff
@@ -16,6 +16,13 @@ class Fio < Formula
 
   uses_from_macos "zlib"
 
+  # Upstream fix for "ld: weak import of symbol '___darwin_check_fd_set_overflow' not supported because of option: -no_weak_imports for architecture x86_64"
+  # https://github.com/axboe/fio/commit/b6a1e63a1.diff
+  patch do
+    url "https://github.com/axboe/fio/commit/b6a1e63a1.diff"
```

Comment:
Audit will complain about this, I think. It probably needs to be
```suggestion
    url "https://github.com/axboe/fio/commit/b6a1e63a1.patch?full_index=1"
```
with the `sha256` adjusted accordingly

Reply:
Ah thanks for pointing the sha is wrong. Just pushed another update.